mmd_tools/operators/misc.py

```diff
@@ -50,27 +50,35 @@ def execute(self, context):
         obj = context.active_object
         root = mmd_model.Model.findRoot(obj)
         if root is None:
-            self.report({ 'ERROR' }, 'Select a MMD model') 
-            return { 'CANCELLED' } 
+            # self.report({ 'ERROR' }, 'Select a MMD model') 
+            # return { 'CANCELLED' }
+            # find all meshes in the active layer
+            act_layer = bpy.context.scene.active_layer
+            meshes_list = [o for o in bpy.data.objects
+                           if o.layers[act_layer] and o.type == 'MESH' and o.mmd_type == 'NONE']
+            active_mesh = meshes_list[0]
```

Comment:
Ooops! I didn't noticed that. The lack of unit tests makes coding harder. Could be possible to use Blender bundled Python to run unit tests? I have not found any information about that on Blender's documentation. 

For the moment I think it's better to just support joining meshes of a MMD model. But it could be good to implement an operator to find existing meshes and attach them to mmd_root. I will work on it and update my changes soon.